src/test/java/core/basesyntax/service/FruitShopServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.functionalityexpansion.ActivityHandlerProvider;
+import core.basesyntax.service.functionalityexpansion.ActivityType;
+import core.basesyntax.service.parsefileinfo.FruitTransactionInfo;
+import core.basesyntax.service.strategy.ActivityHandler;
+import core.basesyntax.service.strategy.BalanceHandler;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class FruitShopServiceTest {
+    private final Storage storage = new Storage();
+    private final ActivityHandlerProvider activityProvider = new ActivityHandlerProvider(storage);
+
+    @Test
+    void execute_NullInput_Ok() {
```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/90136583/1333368a-875f-4dfa-8cbf-84b7e12db6d8)
I think we need to name methoud like this in lower camel case if we are talking about nullInput